README.md

```diff
@@ -1,79 +1,193 @@
-## This is a WIP FAQ for Tusky | Frequently asked questions about the Tusky Mastodon client
+# Tusky Frequently Asked Questions
 
-Which will include some experimentation with tools to write this FAQ and sync it with GitHub, by @mal0ki (current tool is StackEdit).
+## Installing Tusky
 
-## FAQ
-**Question:** When will the latest version of Tusky be available on F-Droid?
+### What Tusky release should I use?
 
-**Answer:** F-Droid works differently than Google Play. We as the developers do not upload anything, F-Droid builds the app from source once we tag a release in our code repository. This is super awesome because you are guaranteed to get the exact code from our repo, but also means it takes longer for releases to be available.
+There are three different releases of Tusky.
 
-##
-**Q:** How does Tusky support push notifications?
+- The mainstream release. Unless you've been told otherwise, this is what you want
+- Beta releases. These are released to testers before every mainstream release
+- Test releases. These are released every time there is a change to Tusky
 
-**A:** We use [Unified Push](https://unifiedpush.org/) to avoid proprietary Google libraries, since we want Tusky to be pure FOSS (free and open-source software) and don't want to send any data to Google.
```

Comment:
Sorry for the nitpick but there shouldn't be a space in Unified Push. It should be UnifiedPush.